Core/Translation/en_EN.json

```diff
@@ -414,7 +417,7 @@
     "refresh": "Refresh",
     "refresh-page": "Refresh page",
     "register-symbol": "#",
-    "renumber-accounting": "Renumber Accounting",
+    "renumber-accounting": "\nRenumber accounting entries...",
```

Comment:
Codacy is warning you that this field called 'renumber-accounting' is duplicated.